css/colourpicker.css

```diff
@@ -36,36 +42,59 @@
   left: 0;
   top: auto;
 }
+
+
 .palette-square .colourpicker-panel {
-  width: 173px;
+  width: 174px;
   height: 152px;
 }
+
+.palette-square.colourpicker-with-alpha .colourpicker-panel {
+  width: 196px;
+}
+
 .palette-limited .colourpicker-panel {
-  width: 181px;
+  width: 196px;
   height: auto;
-  padding: 5px;
+  padding-left: 0px;
+  padding-right: 0px;
+  padding-top: 1px;
+  padding-bottom: 4px;
   background-color: #F8F8F8;
 }
-.colourpicker .cp-list-row {
+
+/*.colourpicker .cp-list-row {
   margin-top: 3px;
-}
+*/}
 .colourpicker .cp-list-row:first-child {
   margin-top: 0;
 }
 .colourpicker .cp-list-col {
+  height: 100%;
+  width: 100%;
+  display:block;
+  /*border: 1px solid #DDDDDD;*/
+}
+
+
+.colourpicker .cp-list-col-outer {
   width: 20px;
   height: 20px;
+  margin-top: 3px;
```

Comment:
if margin left is 4, then margin top should also be 4. It looks a bit off when the vertical and horizontal spacing isn't uniform. Either both at 3 or both at 4.